translations/en-us.yml

```
@@ -253,8 +253,8 @@ move_to_project:
         project: 'Files moved to private projects will no longer be public or discoverable by others.'
         component: 'Files moved to private components will no longer be public or discoverable by others.'
     storage_limit:
-        warning: 'This project is approaching its storage limit.'
-        error: 'This project is past its storage limit.'
+        warning: 'This project is approaching its storage cap.'
+        learn_more: '<a href "https://help.osf.io/hc/en-us/articles/360054528874-OSF-Storage-Caps" data-analytics-name="Learn about storage caps">Learn more about storage limits</a>.'
     file_successfully_moved: 'File was moved successfully!'
     could_not_move_file: 'Could not move file.'
     storage_error: 'This file cannot be added to this project/component because it would exceed the storage limit for OSF Storage.'
```

thinking `has exceeded` since it's an error? That is what the files widget uses
```suggestion
    storage_error: 'This file cannot be added to this project/component because it has exceeded the storage limit for OSF Storage.'
```